features/user_logout.feature

```diff
@@ -1,20 +1,20 @@
 Feature: User Log Out
-    As a user,
-    In order to sign out of my account
-    I would like to have a log out function
+	"As a user,
```

Comment:
keep the indentation consistent in all files

Reply:
I formatted this to use 2 spaces 😳

Reply:
But apparently, it is using 2 x tabs 🤯

Reply:
fixed all files to use x2 spaces per tomas

Reply:
But it is stil 2 x TABS